packages/client/src/Router.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { Layout } from "./components/Layout.js";
+import { HomePage } from "./pages/HomePage.js";
+import { AdminPage } from "./pages/AdminPage.js";
+import { MenuPage } from "./pages/MenuPage.js";
+import { OrderPage } from "./pages/OrderPage.js";
+import { RegistrationPage } from "./pages/RegistrationPage.js";
+
+export const Router = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Layout />} />
+        <Route index element={<HomePage />} />
```

Comment:
This is the only valid route with an index attribute. The router worked overall, but there should only be a single index under each parent route.